Frontend/src/data/customer/inputs.js

```diff
@@ -105,6 +112,10 @@ export default {
       label: "E-Mail",
       group: "Kontakt",
       component: TextInput,
+      props: {
+        required: true,
+        pattern: ".+@.+\.[a-zA-Z]+",
```

Comment:
https://stackoverflow.com/a/201378 :D :D

Reply:
Works :shrug: 😄

Reply:
but yeah. that should do.